test/Microsoft.SqlTools.ServiceLayer.UnitTests/ObjectExplorer/ServerVersionHelperTests.cs

```diff
@@ -151,6 +160,18 @@ private void VerifyCalculateServerType(string serverVersion, SqlServerType expec
             Assert.Equal(expected, actual);
         }
 
+        private void VerifyCalculateServerTypeForEngineEdition(int engineEdition, SqlServerType expected)
+        {
+            ServerInfo serverInfo = new ServerInfo
+            {
+                EngineEditionId = engineEdition
+            };
+
+            SqlServerType actual = ServerVersionHelper.CalculateServerType(serverInfo);
+            
+            Assert.True(actual.ToString().Equals(expected.ToString()), "Verify correct engine edition number");
+        }
```

Comment:
Why are you calling ToString? You should be able to just directly compare with ==